src/assets/translations/en-us.yaml

```diff
@@ -261,6 +261,17 @@ containerRuntime:
       label: containerd
       description: Namespaces for container images; use with nerdctl.
 
+pathManagement:
+  label: Configure PATH
+  tooltip: Choose how you'd like Rancher Desktop to look for Supporting Utilities.
```

Comment:
This tooltip text is kind of placeholder for the time being.. open to recommendations

Reply:
Thanks, I like that a lot. We'll make sure to manage code highlighting along with the PR to fix the description text.